package.json

```diff
@@ -135,11 +140,11 @@
     "react-avatar-editor": "^13.0.0",
     "react-circular-progressbar": "^2.1.0",
     "react-dom": "^18.2.0",
-    "react-native": "0.72.5",
+    "react-native": "^0.73.0",
     "react-native-appstate-hook": "^1.0.6",
     "react-native-drawer-layout": "^3.2.0",
     "react-native-fs": "^2.20.0",
-    "react-native-gesture-handler": "^2.12.1",
+    "react-native-gesture-handler": "~2.14.0",
```

Comment:
Both this and screens needed to be updated to unblock build errors

Reply:
Probably good to run `npx expo install --fix` just so we are at least at the package numbers that v50 wants